src/agri_ai/line_bot/app.py

```
@@ -0,0 +1,253 @@
+"""
+LINE Bot application for Agricultural AI Agent.
+"""
+
+import asyncio
+import logging
+from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)